### PR TITLE
About us

### PR DESCRIPTION
**Pull request by radishevskyi**
# 📝 Description

- added about us section
- added adaptive

# 🔄 Type of change

- [ ] Cleaning up
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality to something)
- [x] Configuration change
- [x] Translation change

# 🧪 How has this been tested / or will it be tested?

- [ ] Manually surfing the app

# ✅ Checklist:

- [x] My changes generate no new warnings in console or terminal
- [x] I have performed a self-review of my code
- [ ] I have updated versions of the package.json and package-lock.json

# 🖼️ Result screenshot/video (optional):
image

# Gif to summarize my emotions (optional):

### CODE REVIEW

File: src/modules/home/about-us/about-us.styles.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  Content: styled(Container)`
+    display: flex;
+    justify-content: space-between;
+    gap: calc(var(--spacing-x3) * 2);
+    min-height: 574px;
+    flex-wrap: wrap;
+  `,
+
+  Div1: styled.div`
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:18]:
```suggestion
ImgColumn: styled.div
```

File: src/modules/home/about-us/about-us.styles.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    position: relative;
+    min-width: calc(50% - var(--spacing-x3));
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:20]: Is this needed? It should work without this.

File: src/modules/home/about-us/about-us.styles.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    top: 10%;
+    right: 27%;
+    z-index: 1;
+  `,
+
+  Logo: styled.img`
+    display: block;
+  `,
+
+  Background1Item: styled.div``,
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:40]: Remove

File: src/modules/home/about-us/about-us.styles.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    position: relative;
+    min-width: calc(50% - var(--spacing-x3));
+    min-height: 100%;
+  `,
+
+  Background1: styled.div`
+    display: flex;
+    align-items: center;
+    position: absolute;
+    background-color: #415346;
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:28]: Isn't it an accent color? If yes or it is similar then use css variable

File: src/modules/home/about-us/about-us.styles.ts [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    display: block;
+  `,
+
+  Background1Item: styled.div``,
+
+  Goals: styled.span`
+    font-size: 30px;
+  `,
+
+  Destroyed: styled.p`
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:46]:
```suggestion
Destroyed: styled.span
```

File: src/modules/home/about-us/about-us.styles.ts [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    flex-wrap: wrap;
+  `,
+
+  Div1: styled.div`
+    position: relative;
+    min-width: calc(50% - var(--spacing-x3));
+    min-height: 100%;
+  `,
+
+  Background1: styled.div`
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:24]:
```suggestion
InfoBlock: styled.div
```

File: src/modules/home/about-us/about-us.styles.ts [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+  Goals: styled.span`
+    font-size: 30px;
+  `,
+
+  Destroyed: styled.p`
+    font-size: 18px;
+  `,
+
+  Background2: styled.div`
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:50]:
```suggestion
ImgBlock: styled.div`
```

File: src/modules/home/about-us/about-us.styles.ts [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    position: absolute;
+    bottom: 0;
+    background-color: white;
+    min-height: 54%;
+    min-width: 54%;
+  `,
+  Background3: styled.div`
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:57]:
```suggestion
ImgBlock: styled.div`
```

File: src/modules/home/about-us/about-us.styles.ts [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+    background-color: white;
+    min-height: 90%;
+    min-width: 43%;
+    position: absolute;
+    top: 0;
+    right: 0;
+  `,
+
+  Div2: styled.div`
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:66]:
```suggestion
TextColumn: styled.div`
```

File: src/modules/home/about-us/about-us.styles.ts [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    gap: var(--spacing-x2);
+    max-width: calc(50% - var(--spacing-x3));
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:72]: Is this needed? It should work without this.

File: src/modules/home/about-us/about-us.styles.ts [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+      rgba(255, 255, 255, 1),
+      rgba(255, 255, 255, 0.2)
+    );
+    background-clip: text;
+    color: transparent;
+    font-weight: var(--font-weight-medium);
+  `,
+
+  AboutText: styled.p`
+    color: #fafafab2;
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:88]: Use same color we have on first screen. And actually whole website use white color for text so it is enough to set white color for text in body. All children elemens will inherit color so there is no need to set white color for every component.

File: src/shared/components/label/label.tsx [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const Label = styled.span`
+  display: block;
+  border: 1px solid rgba(65, 83, 70, 25%);
+  background-color: rgba(65, 83, 70, 20%);
+  border-radius: 20px;
+  color: #979897;
+  padding: var(--spacing) var(--spacing-x2);
+  font-size: 12px;
```
Comment (p-osypov) [inline src/shared/components/label/label.tsx:10]: Use css variable if we have it

**Review by p-osypov — CHANGES_REQUESTED** [review]

**Comment by p-osypov** [comment]
Fill the description and add screenshot of work you made. Desktop and Mobile

File: src/modules/home/about-us/about-us.styles.ts (hunk 4 above)
Comment (radishevskyi) [reply src/modules/home/about-us/about-us.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/5/commits/182f44b38675be57d62bf0cd6bfcd3df0ab32120)

File: src/modules/home/about-us/about-us.styles.ts (hunk 5 above)
Comment (radishevskyi) [reply src/modules/home/about-us/about-us.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/5/commits/182f44b38675be57d62bf0cd6bfcd3df0ab32120)

File: src/modules/home/about-us/about-us.styles.ts (hunk 6 above)
Comment (radishevskyi) [reply src/modules/home/about-us/about-us.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/5/commits/182f44b38675be57d62bf0cd6bfcd3df0ab32120)

File: src/modules/home/about-us/about-us.styles.ts (hunk 7 above)
Comment (radishevskyi) [reply src/modules/home/about-us/about-us.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/5/commits/182f44b38675be57d62bf0cd6bfcd3df0ab32120)

File: src/modules/home/about-us/about-us.styles.ts (hunk 8 above)
Comment (radishevskyi) [reply src/modules/home/about-us/about-us.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/5/commits/182f44b38675be57d62bf0cd6bfcd3df0ab32120)

File: src/modules/home/about-us/about-us.styles.ts (hunk 9 above)
Comment (radishevskyi) [reply src/modules/home/about-us/about-us.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/5/commits/182f44b38675be57d62bf0cd6bfcd3df0ab32120)

File: src/shared/components/label/label.tsx (hunk 12 above)
Comment (radishevskyi) [reply src/shared/components/label/label.tsx]: [fix: styles](https://github.com/p-osypov/army/pull/5/commits/182f44b38675be57d62bf0cd6bfcd3df0ab32120)

File: src/modules/home/first-screen/first-screen.styles.ts [hunk 13]
```diff
@@ -1,44 +1,49 @@
...
+    background-size: cover;
     color: var(--color-white);
+    margin-bottom: 100px;
+    ${mediaWidth('max', 'lg')} {
+      margin-bottom: 70px;
+    }
+    ${mediaWidth('max', 'md')} {
+      margin-bottom: 50px;
+    }
+    ${mediaWidth('max', 'md')} {
```
Comment (p-osypov) [inline src/modules/home/first-screen/first-screen.styles.ts:21]: Remove

**Review by p-osypov — CHANGES_REQUESTED** [review]

File: src/modules/home/first-screen/first-screen.styles.ts (hunk 13 above)
Comment (radishevskyi) [reply src/modules/home/first-screen/first-screen.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/5/commits/5c68731fa194332f8b1d88c2e874c342b0071d23)

**Review by p-osypov — APPROVED** [review]

**Pull request merged**